pkg/docker/util.go

```diff
@@ -363,3 +363,16 @@ func GetDefaultDockerConfig() *api.DockerConfig {
 
 	return cfg
 }
+
+// GetAssembleUser finds an assemble user on the given image.
```

Comment:
Add more doc explaining what is happening here, if cfgAssemble user is not blank ...